File: src/gestorAplicacion/Taquilla/Pelicula.java
/**
 * @author Daniel Alejandro Giraldo Giraldo.
 * @param clase Pelicula.
 * @summary Clase que tiene la imformacion relacionada con las peliculas.
 */

package gestorAplicacion.Taquilla;
import gestorAplicacion.Autocine.Autocine;
import gestorAplicacion.Salas.Sala;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

// clase
public class Pelicula implements Serializable {

    // serializacion
    private static final long serialVersionUID = 1L;
    static List<Pelicula> peliculas;
    static {
        peliculas = new ArrayList<Pelicula>();
    }
    //artributos

    private String nombre;
    private String genero;
    private int duracion;
    private String lenguaje;
    private int clasificacion;
    private Autocine autocine;

// constructor

    public Pelicula(String nombre, String genero, int duracion, String lenguaje, int clasificacion, Autocine autocine) {
        this.nombre = nombre;
        this.genero = genero;
        this.duracion = duracion;
        this.lenguaje = lenguaje;
        this.clasificacion = clasificacion;
        this.autocine = autocine;
        Autocine.agregarPelicula(this);
    }

    // gets y sets
    public String getNombre() {
        return nombre;
    }
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getGenero() {
        return genero;
    }
    public void setGenero(String genero) {
        this.genero = genero;
    }

    public int getDuracion() {
        return duracion;
    }
    public void setDuracion(int duracion) {
        this.duracion = duracion;
    }

    public String getLenguaje() {
        return lenguaje;
    }
    public void setLenguaje(String lenguaje) {
        this.lenguaje = lenguaje;
    }

    public int getClasificacion() {
        return clasificacion;
    }
    public void setClasificacion(int clasificacion) {
        this.clasificacion = clasificacion;
    }
    public static List<Pelicula> getPeliculas() {
        return peliculas;
    }

    public static void setPeliculas(List<Pelicula> peliculas) {
        Pelicula.peliculas = peliculas;
    }
}

File: Practica 2/AutoCine/src/gestorAplicacion/Taquilla/Pelicula.py
#*
# * @author Daniel Alejandro Giraldo Giraldo.
# * @param clase Pelicula.
# * @summary Clase que tiene la imformacion relacionada con las peliculas.
# 

from gestorAplicacion import Autocine.Autocine
from gestorAplicacion import Salas.Sala


# clase
class Pelicula:

    #artributos

    def __init__(self, nombre, genero, duracion, lenguaje, clasificacion, autocine):
        #instance fields found by Java to Python Converter:
        self._nombre = None
        self._genero = None
        self._duracion = 0
        self._lenguaje = None
        self._clasificacion = 0
        self._autocine = None

        self._nombre = nombre
        self._genero = genero
        self._duracion = duracion
        self._lenguaje = lenguaje
        self._clasificacion = clasificacion
        self._autocine = autocine
        gestorAplicacion.Autocine.Autocine.agregarPelicula(self)

    # gets y sets
    def getNombre(self):
        return self._nombre
    def setNombre(self, nombre):
        self._nombre = nombre

    def getGenero(self):
        return self._genero
    def setGenero(self, genero):
        self._genero = genero

    def getDuracion(self):
        return self._duracion
    def setDuracion(self, duracion):
        self._duracion = duracion

    def getLenguaje(self):
        return self._lenguaje
    def setLenguaje(self, lenguaje):
        self._lenguaje = lenguaje

    def getClasificacion(self):
        return self._clasificacion
    def setClasificacion(self, clasificacion):
        self._clasificacion = clasificacion
    @staticmethod
    def getPeliculas():
        return gestorAplicacion.Taquilla.Pelicula.peliculass

    @staticmethod
    def setPeliculas(peliculass):
        Pelicula.peliculass = peliculass